cluster/operations/backup-atc-colocated-web.yml

```diff
@@ -0,0 +1,21 @@
+- type: replace
+  path: /releases/-
+  value:
+    name: backup-and-restore-sdk
+    url: https://bosh.io/d/github.com/cloudfoundry-incubator/backup-and-restore-sdk-release?v=((bbr_sdk_version))
+    sha1: ((bbr_sdk_sha1))
+    version: ((bbr_sdk_version))
+
+- type: replace
+  path: /instance_groups/name=web/jobs/-
+  value:
+    name: bbr-atcdb
+    release: concourse
+    consumes:
+      concourse_db: {from: web}
```

Comment:
An implicit link would've worked, so those lines are technically not needed. If we removed them then bosh would just figure out the implicit linking. If we want less yaml then we can delete these lines. Thoughts?

Reply:
hmmmmm that's interesting - I personally prefer the explicitness that the `from: web` brings - seems like a nice clear way of seeing that the properties are "flowing" from `web` to the `bbr` job.  

Wdyt @YoussB @pivotal-bin-ju ?

Reply:
We removed this because you need a matching provides for the consume.

Reply:
oh, interesting! Thanks!